packages/app/hooks/use-share-nft.ts

```diff
@@ -1,29 +1,49 @@
+import { Linking } from "react-native";
+
 import { useShare } from "app/hooks/use-share";
 import { track } from "app/lib/analytics";
 import { CHAIN_IDENTIFIERS } from "app/lib/constants";
 import { NFT } from "app/types";
 
+import { getTwitterIntent } from "../utilities";
```

Comment:
nit: app/utilities (ocd)

Reply:
Haha it's not a nitpick, no worries. Can we find a way to get absolute imports with auto import on VS Code? I suspect this was added by auto import, right?

Reply:
yeah it's auto import, using absolute paths will solve it!